src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/PreparedStatementTest.java

```diff
@@ -170,6 +170,8 @@ public void testStatementPooling() throws SQLException {
             }
         }
 
+        Utils.skipTestForSQLServer2008((SQLServerConnection) DriverManager.getConnection(connectionString));
```

Comment:
Apparently `RAISERROR` fails too. This might be an actual driver issue. We decided to not merge it until we investigate the test more deeply.